agol_layers.json

```diff
@@ -0,0 +1,26 @@
+{
```

Comment:
I like the idea of moving all of this config to a config file. In the future (post PR merge, in another branch) I think we'll want to add some more metadata tho to make it easier to understand the meaning of each entry (eg, why some have `original_file_name` and some don't.)

Reply:
Yeah, I was realizing that json isn't a good format for comments, etc., so using the pattern in header_mapping.py would probably be wiser.